src/components/ScrollView.md

```diff
@@ -28,7 +28,7 @@ let make = () => {
   <ScrollView ref=scrollViewRef>
```

Comment:
This line should read
```reason
<ScrollView ref=scrollViewRef->Ref.value>
```
Then, `switch (scrollViewRef.current->Js.Nullable.toOption)` works without issues.

@MoOx I also find that syntax somewhat unpleasant, I prefer `React.(scrollViewRef.current)`. Obviously we cannot have a lowercase module name or an uppercase field name, so once the initial shock wears off, it can be understood, but I find it a bit hard to like all the same.